src/components/screens/wallet/send/form/useProcessingSpeed.js

```diff
@@ -0,0 +1,49 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { dynamicFeesRetrieved } from '../../../../../actions/service';
+
+const useProcessingSpeed = () => {
+  const { t } = useTranslation();
+
+  const dispatch = useDispatch();
+  const { dynamicFees } = useSelector(state => state.service);
+  useEffect(() => {
+    dispatch(dynamicFeesRetrieved());
+  }, []);
+  const isLoading = Object.keys(dynamicFees).length === 0;
```

Comment:
The catch is not handled properly and this always stays in loading if the API call fails.